sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/README.md

```diff
@@ -11,9 +11,63 @@ description: Samples for the Azure.Messaging.EventHubs.Processor client library
+
+- **C# 8.0:** The Azure Event Hubs client library makes use of new features that were introduced in C# 8.0.  You can still use the library with older versions of C#, but some of its functionality won't be available.  In order to enable these features, you need to [target .NET Core 3.0](https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-target-frameworks) or [specify the language version](https://docs.microsoft.com/en-gb/dotnet/csharp/language-reference/configure-language-version#override-a-default) you want to use (8.0 or above).  If you are using Visual Studio, versions prior to Visual Studio 2019 are not compatible with the tools needed to build C# 8.0 projects.  Visual Studio 2019, including the free Community edition, can be downloaded [here](https://visualstudio.microsoft.com/vs/).
+
+  **Important Note:** The use of C# 8.0 is mandatory to run the samples without modification.  You can still run the samples if you decide to tweak them.
+
+To quickly create the needed resources in Azure and to receive connection strings for them, you can deploy our sample template by clicking:  
+
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-net%2Fmaster%2Fsdk%2Feventhub%2FAzure.Messaging.EventHubs.Processor%2Fassets%2Fsamples-azure-deploy.json)
+
+## Available samples
+
+- [Hello world](./samples/Sample01_HelloWorld.cs)  
+  An introduction to the Event Processor client, illustrating how to create the client and perform basic operations.
+  
+- [Create an Event Processor client with custom options](./samples/Sample02_ProcessorWithCustomOptions.cs)  
+  An introduction to the Event Processor client, exploring additional options for creating the processor.
+
+- [Perform basic event processing](./samples/Sample03_BasicEventProcessing.cs)  
+  An introduction to the Event Processor client, illustrating how to perform basic event processing.
+
+- [Create checkpoints to track processing state](./samples/Sample04_BasicCheckpointing.cs)  
+  An introduction to the Event Processor client, illustrating how to create simple checkpoints.
+  
+- [Initialize an Event Hub partition for processing by a specific Event Processor client](./samples/Sample05_InitializeAPartition.cs)  
+  An introduction to the Event Processor client, illustrating how to participate in initialization for a partition.
+
+- [Track when an Event Hub partition will no longer be processed by a specific Event Processor client](./samples/Sample06_TrackWhenAPartitionIsClosed.cs)  
+  An introduction to the Event Processor client, illustrating how to track when processing stops for a partition.
+  
+- [Manage the Event Processor when an error is encountered](./samples/Sample07_RestartProcessongOnError.cs)  
```

Comment:
Update the path here as well when filename is corrected (`Processong` => `Processing`).